### PR TITLE
chore: Update the mods section on the P2CE to reflect new mods

### PR DESCRIPTION
**Pull request by AWildErin**
Removed Revolution and added singularity collapse. Currently waiting for some other mods to get back to me and I'll add them too and then I'll mark this finished when that time comes

### CODE REVIEW

File: ChaosInitiative.Web.P2CE/Pages/Mods.cshtml.cs [hunk 1]
```diff
@@ -9,49 +9,72 @@ public class ModsModel : PageModel
         
         public void OnGet()
         {
-            // Filling the list manually for now. This can later be hooked up to control panel if needed
+            // Yes this is bad
+            // No we're not changing it just yet
```
Comment (craftablescience) [inline ChaosInitiative.Web.P2CE/Pages/Mods.cshtml.cs:13]: :trollface:

File: ChaosInitiative.Web.P2CE/Pages/Mods.cshtml.cs [hunk 2]
```diff
@@ -9,49 +9,72 @@ public class ModsModel : PageModel
...
                         {
-                            "Discord",
-                            "https://discord.gg/zrSVsM7"
+                            "Website",
+                            "https://abyssmod.com/"
                         },
                         {
                             "ModDB",
-                            "https://www.moddb.com/mods/portal-revolution-spyce-software"
+                            "http://www.moddb.com/mods/portal-2-abyss"
```
Comment (craftablescience) [inline ChaosInitiative.Web.P2CE/Pages/Mods.cshtml.cs:71]: nit: for PSC, it went "moddb -> discord" but for this one it goes "discord -> website -> moddb"
would be neat if they had the same ordering

Comment (AWildErin) [reply ChaosInitiative.Web.P2CE/Pages/Mods.cshtml.cs:71]: Yep, I can do that!

**Review by craftablescience — APPROVED** [review]
very nice

**Pull request merged**